llvm/test/CodeGen/RISCV/rvv/remat.ll

```diff
@@ -0,0 +1,111 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
```

Comment:
Rematerializable instructions can be hoisted in MachineLICM, need we add some loop tests here?

Reply:
LICM should be able to hoist vid.v even it wasn't rematerializable. Right?

Reply:
Yeah, you're right!